src/v2/ion/RemediationConstants.js

```diff
@@ -42,6 +42,68 @@ const FORMS = {
   TERMINAL: 'terminal',
 };
 
+const FORMNAME_CLASSNAME_MAPPING = {
```

Comment:
Move to a dedicated file like ViewFactory. 
(maybe call it ViewClassNamesFactory ?)